packages/emotion/README.md

```diff
@@ -0,0 +1,17 @@
+# Server-side Rendering
```

Comment:
Should have a generic `# Emotion` header like in our other readmes and this should be a sub header.